cloudproxy-ui/src/App.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <b-container>
+  <div id="app">
+    <b-row>
+      <b-col><h1>CloudProxy</h1></b-col>
+      <b-col cols="12" class="pt-2" md="auto"><b-button @click="reloadPage" variant="success">Refresh</b-button></b-col>
+    </b-row>
+    <br>
+    <div>
+      <b-card no-body>
+        <b-tabs card>
+          <b-tab title="Proxies" active>
+            <ListProxies />
+          </b-tab>
+        </b-tabs>
+
+      </b-card>
+    </div>
+  </div>
+    </b-container>
+</template>
+
+<script>
+import ListProxies from './components/ListProxies.vue'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/claffin/cloudproxy/pullRequest?prid=7329682)
```suggestion
import ListProxies from "./components/ListProxies.vue"
```